Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Android/CrashesDelegate.cs

```diff
@@ -12,7 +12,7 @@ public class CrashesDelegate : AndroidJavaProxy
         public static event Crashes.SendingErrorReportHandler SendingErrorReport;
         public static event Crashes.SentErrorReportHandler SentErrorReport;
         public static event Crashes.FailedToSendErrorReportHandler FailedToSendErrorReport;
-        private static event Crashes.GetErrorAttachmentsHandler GetErrorAttachments;
+        internal static Crashes.GetErrorAttachmentsHandler getErrorAttachmentsHandler = null;
```

Comment:
Why camelCase for something we share like a property (I know we are in C#4 and are limited but that's what we would have used otherwise)? Should be PascalCase.

Reply:
Also no need to `= null`

Reply:
Sure. Copied from another file but will change both to be consistent.